apps/extension/src/routes/popup/settings/settings-advanced.tsx

```diff
@@ -0,0 +1,13 @@
+import { FadeTransition } from '../../../components';
+import { BackIcon } from 'ui/components';
+import { usePopupNav } from '../../../utils/navigate';
+
+export const SettingsAdvanced = () => {
```

Comment:
Unfortunately, this design doesn't quite work so well. 

We have a settings sheet that pops out, but then it links to pages that have completely different backgrounds.

I think we should keep all settings within the sheet. If it is not enough space, we should make the sheet full width.

Reply:
What do you think that when reviewing PR, not to pay an attention to the design, features that we transfer from the previous version? At the moment, we do not have an approved design that I could transfer to our repository, I only transfer the logic from the previous version

Reply:
I suppose it's fine for now, but in general we should always be auditing design decisions that get added to the codebase to make sure it's the highest quality code

Reply:
I understand that, but design issues are not my responsibility until we approve it.